### PR TITLE
Adds dragon hide cloaks

### PR DESCRIPTION
**Pull request by TechnoAlchemisto**
:cl: TechnoAlchemist
rscadd: You can now craft cloaks from the scales of fallen ash drakes, look for the recipe in tablecrafting!
/:cl: 

http://imgur.com/a/NWOER

### CODE REVIEW

**Comment by RandomMarine** [comment]
ruby Forgot something.

**Comment by Incoming5643** [comment]
That changelog is malformed

**Comment by WJohn** [comment]
Those rags do not deserve the title of "dragon hide cloaks".

Sprites need major, major improvements.

**Comment by TechnoAlchemisto** [comment]
Are you volunteering :^)

**Comment by TechnoAlchemisto** [comment]
@Incoming5643 what's wrong with the cl

**Comment by PKPenguin321** [comment]
the first line should be
:cl: YourMcName

but you have no name in it

**Comment by TechnoAlchemisto** [comment]
I have literally never done that and it worked fine, weird

**Comment by KorPhaeron** [comment]
So my biggest hangup with this right now is I'm dumb and made the drakes primary attack fire based, which is already a problem with the fire resistant miners shrugging it off.

So if they get fireproof stuff they can just ignore the drakes after the first one.

But this is my fault for designing the boss around an easily ignorable attack and it feels dumb to hold this PR up over it

**Comment by TechnoAlchemisto** [comment]
The cloak should definitely be immune to ash storms, and the issue is that there is no other way to do that without making it immune to fire too.

**Comment by KorPhaeron** [comment]
Actually there might be a way to do that now that remie made the PR that lets clothing edit mobs

**Comment by KorPhaeron** [comment]
I could make a new ASHPROOF type thing and let the cloak set that

**Comment by TechnoAlchemisto** [comment]
Still doesn't fix the issue of the firesuit explorer suit though, It would make sense for dragon hide to make you fireproof anyhow, you might want to make the dragon do some damage modified lava shit, which isn't accounted for in heat resistance.

**Comment by Jordie0608** [comment]
@PKPenguin321 If left blank the github account name will be used, a name only need be specified there if you want it to be different to that

**Comment by PKPenguin321** [comment]
Oh

**Comment by KorPhaeron** [comment]
Okay I cheated and made ash drakes damage people with their fire breath past fireproofing, will merge this at the same time as that

**Comment by KorPhaeron** [comment]
I'm worried this might kill the incentive to get blue slime potions or get augs but I promised so here goes.

Will probably resprite Soon(TM)

File: code/modules/clothing/suits/cloaks.dm [hunk 1]
```diff
@@ -52,6 +61,27 @@
...
+	allowed = list(/obj/item/device/flashlight,/obj/item/weapon/tank/internals, /obj/item/weapon/resonator, /obj/item/device/mining_scanner, /obj/item/device/t_scanner/adv_mining_scanner, /obj/item/weapon/gun/energy/kinetic_accelerator, /obj/item/weapon/pickaxe, /obj/item/weapon/twohanded/spear)
+	armor = list(melee = 70, bullet = 30, laser = 50, energy = 40, bomb = 70, bio = 60, rad = 50)
+	hooded = 1
+	hoodtype = /obj/item/clothing/head/cloakhood/drake
+	heat_protection = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
+	max_heat_protection_temperature = FIRE_IMMUNITY_SUIT_MAX_TEMP_PROTECT
+	unacidable = 1
+
+/obj/item/clothing/head/cloakhood/drake
+	name = "goliath hide hood"
```
Comment (KorPhaeron) [inline code/modules/clothing/suits/cloaks.dm]: whats the lore behind ash drake cloaks being named goliath hide cloaks?

**Pull request merged**